calculus/integration-partial-fractions.md

```diff
@@ -1 +1,134 @@
 # Partial fraction decomposition
+
+
+Some ODEs give rise to integrals like:
+
+
+\begin{equation}
+y' = \frac{3x+11}{x^2-x-6} \to y = \int \frac{3x+11}{x^2-x-6} \, dx
+\end{equation}
+
+
+
+
+
+This cannot be done with u-substitution... how to take it?
+
+\begin{equation}
+\frac{3x+11}{x^2-x-6} =\frac{3x+11}{(x-3)(x+2)} = \frac{A_1}{x-3} + \frac{A_2}{x+2}
+\end{equation}
+
+ 
+
+\begin{align}
+3x+11 &= A_1(x+2) + A_2(x-3) \\
+&= (A_1 + A_2)x + (2A_1 - 3A_2) \\
+\end{align}
+
+\begin{align}
+ A_1  + A_2 = 3
+\end{align}
+
+\begin{align}
+2A_1 - 3A_2 = 11
+\end{align}
+
+\begin{bmatrix} 1 & 1 & 3\\ 2 & -3 & 11 \end{bmatrix} \to \begin{bmatrix} 1 & 1 & 3\\ 0 & -5 & 5 \end{bmatrix} \to \begin{bmatrix} 1 & 1 & 3\\ 0 & 1 & -1 \end{bmatrix} \to \begin{bmatrix} 1 & 0 & 4\\ 0 & 1 & -1 \end{bmatrix}
+
+\begin{align}
+A_1=4  A_2=-1
+\end{align}
+
+\begin{align}  
+y&=\int\left( \frac{4}{x-3}-\frac{1}{x+2} \right)dx \\ 
+&= 4\ln| x-3 | - \ln|x+2| + C  
+\end{align}
+
+
+## Partial Fraction Decomposition
+
+
+Procedure: For a rational polynomial $ \frac{P\left(x\right)}{Q\left(x\right)}$ where Q is expressed using either linear $\left(ax+b \right)$ or irreducible quadratic $\left(ax^2 + bx + c\right)$ factors and the degree of P is less than Q 
+
+
+
+\begin{align}
+ax+b \to \frac{A}{ax+b}
+\end{align}
+
+\begin{align}
+\left(ax+b \right)^k \to \frac{A_1}{ax+b} + \frac{A_2}{\left(ax+b \right)^2}+ ...+ \frac{A_k}{\left(ax+b \right)^k}
+\end{align}
+
+\begin{align}
+ax^2 + bx + c \to \frac{Ax+B}{ax^2+bx+c}
+\end{align}
+
+\begin{align}
+\left(ax^2 + bx+c \right)^k:repeat like for linear factor
```

Comment:
```suggestion
\left(ax^2 + bx+c \right)^k \text{: repeat like for linear factors}
```
Use `\text` to get text inside an equation to be treated like text instead of part of an expression.